utils/cliutils/commandsflags.go

```diff
@@ -743,6 +744,10 @@ var flagsMap = map[string]cli.Flag{
 		Name:  Overwrite,
 		Usage: "[Default: false] Overwrites the instance configuration if an instance with the same ID already exists.` `",
 	},
+	bpOverwrite: cli.BoolFlag{
+		Name:  Overwrite,
+		Usage: "[Default: false] Overwrites existing build info with build name and number.` `",
```

Comment:
I suggest adding a bit more info about the actual behavior, for example:
```suggestion
		Usage: "[Default: false] Overwrites all existing occurrences of build infos with the provided name and number. Build artifacts will not be deleted.` `",
```